epsagon/trace.py

```diff
@@ -719,6 +719,9 @@ def set_error(self, exception, traceback_data=None):
                 traceback_data = ''.join(
                     traceback.format_list(traceback.extract_stack())
                 )
+        # Convert exception string to Exception type
+        if isinstance(exception, str):
+            exception = Exception(exception)
```

Comment:
Maybe let's define an EpsagonException so user defined set_error(Exception()) won't be aggregated with "string" errors?

not a must

Reply:
Done, good call